lib/rucio/core/rse_counter.py

```diff
@@ -138,6 +139,44 @@ def update_rse_counter(rse_id, session=None):
         update.delete(flush=False, session=session)
 
 
+@transactional_session
+def update_replica_counts(rse_id, session=None):
+    """
+    Read the replica counts and update rse_usage.
+
+    :param rse_id:   The rse_id to update.
+    :param session:  Database session in use.
+    """
+
+    # Get the counds of obsolete and deletably (expired) files
+    obsolete_files, obsolete_bytes = (session.query(func.count(), func.sum(models.RSEFileAssociation.bytes))
+                                      .with_hint(models.RSEFileAssociation,
+                                                 "INDEX_FFS(REPLICAS REPLICAS_TOMBSTONE_IDX)", 'oracle')
```

Comment:
This index is a function based index. I'm not sure it will be used like this. You can have a look how it is done for the reaper:
https://github.com/rucio/rucio/blob/55ddd574709092220f0ae70911e0964538863c46/lib/rucio/core/replica.py#L1718